perf/sds-tests/sds-cert-rotation-citadel/run-test.sh

```diff
@@ -0,0 +1,54 @@
+# Create a cluster for testing
+# Set PROJECT as the name of your GCP project
+export PROJECT="endpoints-jenkins"
+# Set CLUSTER as the name of your newly created GKE cluster 
+export CLUSTER="istio-cluster-test-sds-enabled"
+# Set ZONE as the zone of your newly created GKE cluster 
```

Comment:
Setup istio is a separate step. Just refer to it, or call it.

Reply:
This file has been removed; the test is integrated with other scripts in this repo.